### PR TITLE
Update configuration to match version 2.1.1

### PR DESCRIPTION
**Pull request by veger**
I did the following:

* grabbed /etc/graylog/server/server.conf from the Debian package (version 2.1.1-2) and matched the _graylog.server.conf.j2_ to it.
* removed unused defaults and added the new ones.
* updated _graylog.server.elasticsearch.yml.j2_  since `graylog_elasticsearch_node_name` got replaced by `elasticsearch_node_name_prefix` (see http://docs.graylog.org/en/2.1/pages/upgrade/graylog-2.0.html).

### CODE REVIEW

**Comment by danvaida** [comment]
@veger can you please adhere to the standards of the repo regarding the syntax? see reasoning here https://github.com/Graylog2/graylog-ansible-role/pull/47#issuecomment-255474841

**Comment by mariussturm** [comment]
@danvaida could you please point to one or two lines with syntax validations as examples. I want to merge this soon.

**Comment by veger** [comment]
I updated the `# Web UI` part of _defaults/main.yml_

If there are other issues, either with formatting or something else, please let me know

File: defaults/main.yml [hunk 1]
```diff
@@ -103,6 +103,11 @@ graylog_http_write_timeout:                            '10s'
 graylog_ldap_connection_timeout:                       2000
 graylog_dashboard_widget_default_cache_time:           '10s'
 
+# Content packs
+graylog_content_packs_loader_enabled: true
```
Comment (mariussturm) [inline defaults/main.yml]: Upper case

File: defaults/main.yml [hunk 2]
```diff
@@ -103,6 +103,11 @@ graylog_http_write_timeout:                            '10s'
...
+graylog_content_packs_dir: /usr/share/graylog-server/contentpacks
```
Comment (mariussturm) [inline defaults/main.yml]: single quotes

File: defaults/main.yml [hunk 3]
```diff
@@ -136,28 +140,29 @@ graylog_transport_email_from_email:        ''
...
 # Web UI
-graylog_web_listen_uri:       'http://0.0.0.0:9000/'
-graylog_web_endpoint_uri:     ''
-graylog_enable_cors:          'False'
-graylog_enable_gzip:          'False'
-graylog_web_enable_tls:       'False'
-graylog_web_tls_cert_file:    ''
-graylog_web_tls_key_file:     ''
-graylog_web_tls_key_password: ''
+graylog_web_enable:                  True
```
Comment (mariussturm) [inline defaults/main.yml]: single quotes

**Comment by veger** [comment]
Updated your mentioned syntax errors, including some additional/similar, as a separate commit (for reviewing purposes).
If you want to have the commits combined, please let me know.

**Comment by mariussturm** [comment]
Ok thanks! The idea was to quote True/False when it's used in a template to signal that it will re rendered somewhere but when it's used internally it's not. Like the install switches, will fix that.

**Comment by veger** [comment]
Ah... now I understand the reasoning/difference. Thanks

**Pull request merged**